library/unittest.mock.po

```diff
@@ -6,28 +6,32 @@ msgstr ""
 "Project-Id-Version: Python 3.6\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2019-03-20 09:03+0100\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2019-04-02 23:51+0200\n"
 "Language-Team: FRENCH <traductions@lists.afpy.org>\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Bousquié Pierre <pierre.bousquie@gmail.com>\n"
+"X-Generator: Poedit 2.2.1\n"
 
 #: ../Doc/library/unittest.mock.rst:3
 msgid ":mod:`unittest.mock` --- mock object library"
-msgstr ""
+msgstr ":mod:`unittest.mock` --- Bibliothèque d'objets simulacres"
```

Comment:
```suggestion
msgstr ":mod:`unittest.mock` — Bibliothèque d'objets simulacres"
```

C'est surprenant, je sais, alors j'ai écrit un paragraphe à ce sujet dans le README : https://github.com/python/python-docs-fr#the-case-of----

Aussi : https://github.com/python/python-docs-fr#how-to-input-em-dash-ellipsis-french-quotes-or-no-break-spaces

Reply:
C'est toujours plus classe :)

Reply:
Je suis tenté de faire un petit script qui vas bien pour ce genre de chose... reste a voir s'il existe une lib pour traiter les po et ne toucher que les parties traduites.

Reply:
J'utilise pypi.org/p/polib pour ça parfois, c.f. https://pypi.org/project/poautofill/ et https://pypi.org/p/pomerge/

Reply:
Ça mériterai presque de se faire une commande qui prend un pattern et un remplacement àla :

    posed 's/---/—/' **/*.po

comme ça on pourrait l'utiliser pour plein d'autres fix (une faute d'orthographe qui revient souvent ou ce genre de chose).

Attention en passant, polib ne wrap pas correctement, il faut passer du powrap derrière.

Reply:
@Krilivye ping 😉